src/client/assets/icons/ftmscan.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M20.7123 0H3.28767C1.47194 0 0 1.47194 0 3.28767V20.7123C0 22.5281 1.47194 24 3.28767 24H20.7123C22.5281 24 24 22.5281 24 20.7123V3.28767C24 1.47194 22.5281 0 20.7123 0Z" fill="black"/>
```

Comment:
same "remove fill="black" so we can change icon color in different themes with css"

Reply:
removed "fill=black" but can't see fantom icon in themes that change icons to white. Seen in the following themes:

- Mono Dark
- Classic V2

Reply:
@TacosTonight you will need to add fill: {yourColor} in the css of the icon

Reply:
in this case, IconScan